src/NServiceBus.Msmq.AcceptanceTests/When_setting_label_generator.cs

```diff
@@ -82,35 +78,18 @@ public Endpoint()
                 });
             }
 
-            public class StartHandler : IWantToRunWhenBusStartsAndStops
-            {
-                public Context Context { get; set; }
-                public ReadOnlySettings Settings { get; set; }
-
-                public Task Start(IMessageSession session)
-                {
-                    Context.GeneratorWasCalled = Settings.Get<bool>("GeneratorWasCalled");
-                    return Task.FromResult(0);
-                }
+            static Context Context { get; set; }
 
-                public Task Stop(IMessageSession session)
-                {
-                    return Task.FromResult(0);
-                }
+            public void Run(SettingsHolder config)
```

Comment:
Pretty sure @SimonCropp had to do it for a good reason (that I can't remeber ), simon?